scripts/meeting_visit_count.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import datetime
+import logging
+import sys
+import requests
+from modularodm import Q
+
+from website import settings
+from website.app import init_app
+from website.models import Tag, Conference
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def main(dry_run=True):
```

Comment:
The script would still need to do what it's doing, and would also need to update a field on nodes submitted to any conference, but would only have to be run once. I have little experience with piwik, but it seems like we would want to limit the number of times we have to make that request for all Conference-type nodes.